src/devices/Common/Iot/Device/NumberTool.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device
+{
+    /// <summary>
+    /// Helpers for number.
+    /// </summary>
+    internal static class NumberTool
```

Comment:
optional: I'd prefer if we called it something like NumberHelpers or ConversionHelpers or something along these lines but since it's internal I'm fine with this as is. I guess the "Tool" is the most confusing part of the name for me since tool sounds like a standalone application to me